README.md

```diff
@@ -62,12 +62,25 @@ cmake --build ./build -j $(nproc)
 CTEST_PARALLEL_LEVEL=$(nproc) make -C build test
 ```
 
-
 ## Have a Question?
 If you have any questions about Submitting PR's, Opening Issues, s2n-tls API usage, or something similar, we have a public chatroom available here to answer your questions: https://gitter.im/awslabs/s2n
 
 Otherwise, if you think you might have found a security impacting issue, please instead follow [our Security Notification Process.](#security-issue-notifications)
 
+## Documentation
+
+s2n-tls uses [Doxygen](https://doxygen.nl/index.html) to document it's public API. The latest s2n-tls documentation can be found on [GitHub pages](https://aws.github.io/s2n-tls/doxygen/). This URL tracks the `main` branch.  Older versions of s2n-tls can have their API Documentation generated locally with Doxygen.
+
+To generate the doxygen locally, install doxygen and run `doxygen docs/doxygen/Doxyfile`. The doxygen documentation can now be found at `docs/doxygen/output/html/index.html`.
+
+Doxygen can be installed via one of the following platform dependent commands. Alternatively, more instructions are available at the [Doxygen](https://doxygen.nl/download.html) webpage.
+
+```sh
+brew install doxygen # For MacOS
+apt install doxygen # For Debian based distros
+yum install doxygen # For Amazon Linux 2
+```
```

Comment:
Do we want to own this information? Should we maybe only point people towards the Doxygen documentation?

Reply:
I'll remove this to make the instructions terser. I've already included a link to the website's installation so I agree that should be enough